packages/teams-js/src/public/meeting.ts

```diff
@@ -334,6 +334,47 @@ export namespace meeting {
      * @returns A promise with the updated microphone state
      */
     micMuteStateChangedCallback: (micState: MicState) => Promise<MicState>;
+    /**
+     * Callback for the host to tell the app to change its speaker selection
+     */
+    audioDeviceSelectionChangedCallback?: (selectedDevices: AudioDeviceSelection) => void;
+  }
+
+  /**
+   * Interface for AudioDeviceSelection from host selection
+   *
+   * @hidden
+   * Hide from docs.
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   *
+   * @beta
+   */
+  export interface AudioDeviceSelection {
+    speaker?: AudioDeviceInfo;
+    microphone?: AudioDeviceInfo;
```

Comment:
Since you said in the other PR that it's valid to somethings have no error, no speaker, and no microphone, can you update the documentation so that developers will know how to handle that case? I believe you said they should get default devices instead?

Reply:
Added